libnavui-dropin/src/main/res/values-night/colors.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="mapbox_button_fg_color">@color/colorOnSurface</color>
-    <color name="mapbox_button_bg_color">@color/colorSurface</color>
+    <color name="mapbox_drop_in_primary">@color/colorSecondary</color>
+    <color name="mapbox_drop_in_surface">#fff</color>
+    <color name="mapbox_drop_in_onSurface">#3C3E3E</color>
```

Comment:
Similar to above. Reason being theme is going to be controlled by NSS. Let's not introduce new color key_value pairs and rather keep using the ones we have so it's easy to just change them.

Same for all the other colors below